app/src/androidTest/java/com/github/se/studybuddies/database/FakeData.kt

```diff
@@ -76,20 +84,35 @@ val fakeGroup1 =
         timerState = TimerState(0L, false),
     )
 
+val fakeGroup2 =
```

Comment:
I think it is better to not  use harcoded strings => put them in strings.xml you can do that in another pr